sqlmesh/magics.py

```diff
@@ -618,16 +618,49 @@ def rewrite(self, context: Context, line: str, sql: str) -> None:
         help="Transpile project models to the specified dialect.",
     )
     @argument(
-        "--new-line",
+        "--newline",
         action="store_true",
         help="The output dialect of the sql string.",
     )
+    @argument(
+        "--normalize",
+        action="store_true",
+        help="Whether or not to normalize identifiers to lowercase.",
+    )
+    @argument(
+        "--pad",
+        type=int,
+        help="Determines the pad size in a formatted string.",
+        default=2,
+    )
+    @argument(
+        "--indent",
+        type=int,
+        help="Determines the indentation size in a formatted string.",
+        default=2,
+    )
+    @argument(
+        "--normalize-functions",
+        type=str,
+        help="Whether or not to normalize all function names. Possible values are: 'upper', 'lower'",
+    )
+    @argument(
+        "--leading-comma",
+        action="store_true",
+        help="Determines whether or not the comma is leading or trailing in select expressions. Default is trailing.",
+    )
+    @argument(
+        "--max-text-width",
+        type=int,
+        help="The max number of characters in a segment before creating new lines in pretty mode.",
+        default=80,
+    )
     @line_magic
     @pass_sqlmesh_context
     def format(self, context: Context, line: str) -> None:
         """Format all SQL models."""
         args = parse_argstring(self.format, line)
-        context.format(args.transpile, args.new_line)
+        context.format(**vars(args))
```

Comment:
what's vars?

Reply:
The arg parser returns a Namespace. `vars` accesses the `__dict__` attribute basically. On namespaces, the is well defined behavior to convert it to a dict.

Reply:
See stdlib docs mention the same

https://docs.python.org/3/library/argparse.html#the-namespace-object